readme.md

```diff
@@ -1,3 +1,42 @@
 # Product Owner Guide
 
 A rough guide for those who are product owners on a dwyl project.
+
+## What is a Product Owner?
+
+**A Product Owner (PO)** is someone who represents the views and needs of the
+target audience, stakeholders and the business they represent in agile scrum
+methodology. They must understand and communicate the vision of the product to
+the rest of the team.
+
+They are the liaison between customers and the development team. This means that
+they are responsible for working with stakeholders and user groups to determine
+their needs. Then through collaboration with the Scrum Master and developer team
+they prioritise and discuss these ideas moulding them into features to be built.
+
+**PO Responsibilities:**
+1. Owner and creator of the product backlog* :crown:
+2. Adding new features to the backlog. :new:
+3. Approving or critiquing development work for release. :white_check_mark:
+4. Clearly define, understand and _empathise_ with the end-user. 👩 💡
+5. Involving customers or users and getting their feedback to continuously shape
+the product development. :+1: :thumbsdown:
+6. Maintaining and _updating_ the backlog: adjusting the features and _priority_
+as needed after each sprint or iteration. :small_red_triangle: :small_red_triangle_down:
+7. Describing the user-persona(s) in as much detail as possible to make the
+"pain" they feel _real_ (_especially useful when the team is remote and don't
+often meet with the end-users_) 📝
+8. Attach a _business_ or _customer_ value to each story/task in the backlog to
+ensure that we are building features that are genuinely wanted/needed by users. 💸
+9. Have a clear picture of "competitors" and the "market" for similar
+products/service to avoid "re-inventing" something that already exists! (_i.e.
+  don't try to replace App/Site ABC, you are wasting your time! instead find
+  something they aren't doing or are doing poorly and focus on that!_) 🔍
+10. Be familiar with _existing_ UX/UI through personal use of other/existing
+Apps/services to know what can be done/re-used. ♻️
+11. Trust that your technical team is knowledgeable/skilful/motivated and does not
+need "baby sitting" ... 🙄
```

Comment:
😆 Maybe put this in a more politically correct way that doesn't make it sound like we think all POs are a thorn in our sides? 🙈